src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/GroupBox.xaml

```diff
@@ -0,0 +1,43 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+
+    <Thickness x:Key="GroupBoxPadding">0,0,0,15</Thickness>
+    <system:Double x:Key="GroupBoxHeaderFontSize">20</system:Double>
+    <Thickness x:Key="GroupBoxHeaderMargin">0,4,0,20</Thickness>
+
+    <Style x:Key="DefaultGroupBoxStyle" TargetType="{x:Type GroupBox}">
+        <Setter Property="OverridesDefaultStyle" Value="True" />
+        <Setter Property="Foreground" Value="{DynamicResource GroupBoxForeground}" />
+        <Setter Property="Padding" Value="{StaticResource GroupBoxPadding}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type GroupBox}">
+                    <Grid
+                        Background="{TemplateBinding Background}"
```

Comment:
Missing TemplateBinding for the following properties : 
- BorderThickness
- BorderBrush